mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -756,13 +728,18 @@ ::mediapipe::Status GlAnimationOverlayCalculator::GlSetup() {
     // generate ambient lighting of the scene on the object. Range is [0.0-1.0],
     // with the factor being proportional to the brightness of the lighting in the
     // scene being applied to the object
-    const float kAmbientLighting = 0.95;
+    const float kAmbientLighting = 0.9;
```

Comment:
nit: this is very high, so you'll probably want to lower this so you can see the effect of the lighting better.  Also, in particular we will want to have kAmbient + kLightColor <= 1.0, because otherwise we will be potentially overflowing the normal [0,1] color bounds, so we can see clamping behavior.